src/debuginfo.cpp

```diff
@@ -734,7 +734,7 @@ static void get_function_name_and_base(const object::ObjectFile *object, bool in
         saddr = nullptr;
     // Try platform specific methods first since they are usually faster
     if (saddr && !*saddr) {
-#if defined(_OS_LINUX_) && !defined(JL_DISABLE_LIBUNWIND)
+#if (defined(_OS_LINUX_) || defined(_OS_FREEBSD_)) && !defined(JL_DISABLE_LIBUNWIND)
```

Comment:
I just print `*saddr` in (https://github.com/JuliaLang/julia/pull/22547/files/2ca716c3bb406b99a26ef4e556f4b41455ae93e6#diff-d4b80bf89e46982070f234dd7cc344a9R740)
and it did get something.
(Not strictly verified the source of libunwind